Agilefreaks/FreaksCatalog-API
# [121] Add projects to Freak

## Description
Create a Project model with `name`, `description`, `logoUrl`, `freaks` and `technologies` properties and tables needed for this model and his associations: `projects` table for projects, `freaks_projects` for many-to-many relation between freaks and projects, `projects_technologies` for many-to-many relationship between projects and technologies.
Also, the json data for projects was removed, now when we run the `freaks_spec` test a project object is created with help of factories.

## Related

[[121] Add Projects to Freak](https://trello.com/c/RQBdanH4/121-add-projects-to-freak)

## Steps to Test or Reproduce
The `seeds.rb` file will populate tables `projects`, `projects_technologies` and `freaks_projects` with some data. You can run next GraphQl query to get all freaks and see that data for projects will be taken from database.
```GraphQl
query Freaks($last: Int, $before: String) {
  freaks(last: $last, before: $before) {
    pageInfo {
      endCursor
      startCursor
      hasPreviousPage
      hasNextPage
    }
    edges {
      node {
        projects {
          id
          name
          description
          logoUrl {
            id
            uri
          }
          freaks {
            id
            firstName
          }
          technologies {
            id
            name
            description
          }
        }
      }
    }
  }
}

```
